### PR TITLE
converting keras only makes model.json

### PR DESCRIPTION
**Pull request by smsaladi**
When converting a keras model, a single file `model.json` file as well as binary weight files are produced. It would be very helpful to state this clearly (since it tripped me up) though my wording in the PR is probably less than ideal.

More details in an issue here: tensorflow/tfjs#780

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-converter/222)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,10 +105,11 @@ $ tensorflowjs_converter \
 
 ### Web-friendly format
 
-The conversion script above produces 3 types of files:
+The conversion script above produces 4 types of files:
 
 * `tensorflowjs_model.pb` (the dataflow graph)
 * `weights_manifest.json` (weight manifest file)
+* `model.json` (the two above, in a single file)
```
Comment (pyu10055) [inline README.md]: Thanks, you may want to identify that for Keras model, converter generates model.json and weight sharding file.
for other format it generates the other three.

Comment (smsaladi) [reply README.md]: Agreed!

**Comment by smsaladi** [comment]
[skip ci]

File: README.md [hunk 2]
```diff
@@ -105,12 +105,16 @@ $ tensorflowjs_converter \
...
 * `group1-shard\*of\*` (collection of binary weight files)
 
+For `keras` input files, the converter generates `model.json` and sharded weight files.
+For other input formats, it generates the other three.
```
Comment (pyu10055) [inline README.md]: it would be good to identify what the other three files are.

File: README.md [hunk 3]
```diff
@@ -112,8 +112,8 @@ The conversion script above produces 4 types of files:
 * `model.json` (the two above, in a single file)
 * `group1-shard\*of\*` (collection of binary weight files)
 
-For `keras` input files, the converter generates `model.json` and sharded weight files.
-For other input formats, it generates the other three.
+For `keras` input files, the converter generates `model.json` and `group1-shard\*of\*`.
+For other input formats, it generates the `tensorflowjs_model.pb`, `weights_manifest.json`, and `group1-shard\*of\*`.
```
Comment (smsaladi) [inline README.md]: What do you think about this?

**Pull request merged**